client/reader/_style.scss

```diff
@@ -17,6 +17,22 @@
 	}
 }
 
+.reader__content,
+.reader__full-post-content {
+	sup, sub {
+		position: relative;
+		font-size: 0.83em;
```

Comment:
I went with 0.83em because that's what's in the default stylesheet for most browsers as far as I know (sorry I can't find a more recent table of defaults, this is a bit vintage):

https://lists.w3.org/Archives/Public/www-style/2008Jul/att-0124/defaultstyles.htm